browser/content_settings/BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//brave/components/brave_shields/browser/buildflags/buildflags.gni")
+
+source_set("content_settings") {
+  if (brave_stp_enabled) {
```

Comment:
even though this is currently only used for stp, I don't think we need to keep it behind `brave_stp_enabled` because nothing it calls is behind the `BRAVE_STP_ENABLED` flag. We actually want these build flags to be internalized as much as possible in the component so the calling code doesn't have to sort out what needs to be guarded and what doesn't.